R/FilterState.R

```diff
@@ -602,14 +602,18 @@ FilterState <- R6::R6Class( # nolint
     # @param dataname `character(1)` name of data set to prepend to variables
     # @return a `call`
     add_keep_na_call = function(filter_call, dataname) {
-      if (isTRUE(private$get_keep_na())) {
+      if (isTRUE(private$get_keep_na()) & length(filter_call) > 0) {
```

Comment:
I prefer `!is.null(filter_call)` over `length(filter_call) > 0`.

Reply:
3df261712f9e191ffbea8651259c6c1d0fbde195